### PR TITLE
Projects

### PR DESCRIPTION
**Pull request by hiimchrislim**
Add Projects page progress so far

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,6 +17,8 @@
     "lint": "yarn lint:js && yarn lint:style"
   },
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.30",
```
Comment (ArsalaBangash) [inline package.json:20]: I don't recommend pulling all of fontawesome so we can get the fa-play icon

File: components/ProjectCard.vue [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        target="_blank"
+      >Demo
+      </b-button>
+      <b-button
+        size="lg"
+        class="button"
+        :href="project.demo"
+        target="_blank"
+      >
+        <fa-icon
```
Comment (ArsalaBangash) [inline components/ProjectCard.vue:39]: I don't think we should pull fontawesome just for this play icon. Let's grab an svg

File: components/ProjectCard.vue [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+          class="mr-2"
+          :icon="['fas', 'play']"
+        />Docs
```
Comment (ArsalaBangash) [inline components/ProjectCard.vue:42]: Let's remove docs and only keep repo and demo

**Review by ArsalaBangash — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (hiimchrislim) [reply package.json:20]: Alright

**Comment by hiimchrislim** [comment]
Just restoring branch for demo

**Pull request merged**